runtime/src/main/java/org/corfudb/runtime/object/StreamViewSMRAdapter.java

```diff
@@ -124,7 +123,8 @@ public void gc(long trimMark) {
             }
             data = streamView.previous();
         }
-        return null;
+
+        return new ArrayList<>();
```

Comment:
Can you use `Collections.emptyList()` instead?

Reply:
done